### ISSUE TITLE
About train

### ISSUE THREAD

**Issue by ConanMusk** [XPixelGroup/DiffBIR#54]
![image](https://github.com/XPixelGroup/DiffBIR/assets/67884112/6d3fc72b-e0ce-4c88-8065-e9137145da2e)
 I want to know what is the parameter of hq_dir_path and validation_set_size? Can you illustrate a specific example? I'd appreciate it if you'd get back to me.

**Comment by 0x3f3f3f3fun** [comment]
Hi! `hq_dir_path` is the directory path of high-quality images (ground truth images), we use "imagenet1k/train/" as hq_dir_path. All images found in `hq_dir_path` will be splited into training set and validation set and you can decide the size of validation set by setting `validation_set_size`.
